mkdocs.yml

```diff
@@ -0,0 +1,50 @@
+site_name : Img AI App
+
+# theme: readthedocs
+theme:
+  name : material
+
+nav:
+  - Home Page : 'index.md'
+  - Open Source :
+    - Programs we are part of :
+      - info : '#'
```

Comment:
These things make no sense. Please see the deploy the website yourself and understand logically what is happening